src/components/gui/gui.css

```diff
@@ -148,3 +149,17 @@
     height: $stage-menu-height;
     padding: $space;
 }
+
+.extensionButton {
```

Comment:
If you make that change I mentioned below about the `classNames` helper, it'll also automatically convert camel case to skewer case used in css, so this would become `.extension-button`

Reply:
I was wondering about that... thanks!